### DISCUSSION TITLE
Activate Transaction links on `Optional fields for transactions`

### DISCUSSION THREAD

**Discussion by danielnetop** [firefly-iii/firefly-iii#5513] [Firefly III] [answered]
Hi, I've started to use transaction links more often and I've just saw that we do have the transaction links to be showed optionally on the preferences page, however I'm not noticing any difference on the creation form, was it supposed to show, or is this something that was going to be implemented in the v2/v3 layout?

(I've just noticed my two open tickets on this project we're related with transaction links 🤦, and I was just about opening another one regarding this but decided to ask here first in case it's not an issue but simply a "future feature"

**Comment by JC5** [comment, marked as answer]
There is no difference, I'm not sure why I added it.

**Reply by danielnetop** [reply]
No worries, I was just curious as if I was thinking the right thing or not.
Keep up the awesome work you've been doing 👍
